packages/cli/README.md

```diff
@@ -476,6 +485,22 @@ OPTIONS
   -m, --metadata=metadata  Optional metadata for the stream
 ```
 
+### `glaze tile:show STREAMID SYNC_OPTION`
```

Comment:
was this just missing from the readme entirely before?

Reply:
Yes